backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/dto/DeleteRepresentationSuccessPayload.java

```diff
@@ -29,21 +30,22 @@
 @GraphQLObjectType
 public final class DeleteRepresentationSuccessPayload implements IPayload {
 
-    private final Project project;
+    private final UUID representationId;
 
-    public DeleteRepresentationSuccessPayload(Project project) {
-        this.project = Objects.requireNonNull(project);
+    public DeleteRepresentationSuccessPayload(UUID representationId) {
+        this.representationId = Objects.requireNonNull(representationId);
     }
 
+    @GraphQLID
     @GraphQLField
     @GraphQLNonNull
-    public Project getproject() {
-        return this.project;
+    public UUID getRepresentationId() {
+        return this.representationId;
     }
 
     @Override
     public String toString() {
-        String pattern = "{0} '{'project: '{'id: {1}, name: {2}'}''}'"; //$NON-NLS-1$
-        return MessageFormat.format(pattern, this.getClass().getSimpleName(), this.project.getId(), this.project.getName());
+        String pattern = "{0} '{'representationId: {1}'}'"; //$NON-NLS-1$
+        return MessageFormat.format(pattern, this.getClass().getSimpleName(), this.representationId);
     }
 }
```

Comment:
I believe the previous implementation was done like it was because of rule we decided (and documented in an ADR) that this kind of mutations should return the enclosing "context" (parent element). The rule is still followed (at least as of this commit) by e.g. `DeleteDocumentSuccessPayload` (returns the `Project`), `DeleteProjectSuccessPayload` (returns the `IViewer`), `DeleteObjectSuccessPayload` (returns the `Document`) and maybe others.

Reply:
I completely agree with your comment, this should be returning the enclosing parent element. I have no idea how / what this parent element should look like when we will be done with the integration of modelers so for now, I'm just retuning this. I would prefer keeping our original philosophy since it's better, I just don't know how I could respect it for now